test/test-utils.test.js

```diff
@@ -0,0 +1,103 @@
+"use strict";
+
+const { extractHash } = require("./testUtils");
+
+describe("extractHash functionality", () => {
+	it("should throw Error if there is empty string", () => {
+		let stdout = "";
```

Comment:
any reason for using let instead of const?

Reply:
I could not find any usecase of `const` here. Can you suggest some so that I can understand the change better?

Reply:
Conventionally some people you use `let` when the variable gets reassigned. If it doesn't, `const` it's the default choice

Reply:
@ematipico @evenstensberg used const wherever needed.